### ISSUE TITLE
If the quality is set after going fullscreen, the control bar will stay visible

### ISSUE THREAD

**Issue by TheFrenchGhosty** [omarroth/invidious#1263]
When using the "Invidious" theme for the player, if the user change the quality (using dash) after going full-screen the control bar will always stay visible, requiring a refresh of the page.

The only way to have a specific quality AND the video in full-screen is to set the quality, then go full-screen.

**Comment by Randshot** [comment]
@TheFrenchGhosty While not ideal, paste the following into the browser console, and then move the mouse over the player, which should fix the control bar:
```js
player.options_.inactivityTimeout = 100
```

**Comment by WaywardHeart** [comment]
Videojs fixed this in videojs/video.js#7383, which landed in v7.15.1.

**Comment by foxjaw** [comment]
Is this progressing btw ? I'm keen to look into this issue.

**Comment by unixfox** [comment]
No. We are stuck in old videojs version due to https://github.com/iv-org/invidious/issues/2848
